experimental/dds/tree2/src/test/simple-tree/list.spec.ts

```diff
@@ -56,25 +39,24 @@ describe("List", () => {
 		return Array.from({ length }, (_, i) => String.fromCodePoint(0x41 + i));
 	}
 
-	/** Helper that creates a new SharedTree with the test schema and returns the root proxy. */
-	function createTree(): TreeField<typeof schema.rootFieldSchema> {
-		return getOldRoot(schema, { numbers: [], strings: [] });
-	}
-
 	// TODO: Combine createList helpers once we unbox unions.
 	/** Helper that creates a new List<number> proxy */
 	function createNumberList(items: readonly number[]) {
-		const list = createTree().numbers;
-		list.insertAtStart(...items);
+		const list = getRoot(
+			(_) => _.list(_.number),
+			() => items,
+		);
 		assert.deepEqual(list, items);
 		return list;
 	}
 
 	// TODO: Combine createList helpers once we unbox unions.
```

Comment:
Has this been done yet? Can we dedupe these helper functions?

Reply:
I removed the stale TODOs, thanks.

(Unions now work, but combining these helpers is no longer as interesting now that we can easily conjure a SharedTree with any schema from thin air in a couple lines of code.)